pkg/serverless/aws/arn.go

```diff
@@ -115,3 +160,89 @@ func RestoreCurrentStateFromFile() error {
 	SetRequestID(restoredState.CurrentReqID)
 	return nil
 }
+
+// BuildFunctionARNFromEnv reconstructs the function arn from what's available
+// in the environment.
+func BuildFunctionARNFromEnv() string {
+	region := os.Getenv(regionEnvVar)
+	functionName := os.Getenv(functionNameEnvVar)
+	alias := os.Getenv(aliasEnvVar)
+	accountID := fetchAccountID()
+	arnPrefix := "arn:aws"
```

Comment:
Nit: Technically, I think the prefix is just `arn:`. The thing that comes after it is the "partition". Maybe we could separate those things into a separate constant/variable.

https://docs.aws.amazon.com/quicksight/latest/APIReference/qs-arn-format.html